src/display/text_layer.js

```diff
@@ -326,7 +331,11 @@ class TextLayer {
       divStyle.left = `${scaleFactorStr}${left.toFixed(2)}px)`;
       divStyle.top = `${scaleFactorStr}${top.toFixed(2)}px)`;
     }
-    divStyle.fontSize = `${scaleFactorStr}${fontHeight.toFixed(2)}px)`;
+    // We multiply the font size by #getMinFontSize(), and then #layout will
+    // scale the element by 1/#getMinFontSize(). This allows us to effectively
+    // ignore the minimum font size enforced by the browser, so that the text
+    // layer <span>s can always match the size of the text in the canvas.
+    divStyle.fontSize = `${scaleFactorStr}${(TextLayer.#getMinFontSize() * fontHeight).toFixed(2)}px)`;
```

Comment:
Any reason to not use the already cached value here?
```suggestion
    divStyle.fontSize = `${scaleFactorStr}${(this.#layoutTextParams.minFontSize * fontHeight).toFixed(2)}px)`;
```

Reply:
This made me realize that, given that `TextLayer.#getMinFontSize()` always returns the same value anyway, I don't even need to store it in `this.#layoutTextParams`.

With this, plus your suggestion in the other comment to just compute it in the constructor, I simplified a big how we access the computed `minFontSize` to just always grab it from `TextLayer.#minFontSize`.